functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,32 @@
 #include "impl.h"
 
+BeerBrand BeerOrganizer::operator()()
+{
+    BeerBrand bb = BeerBrand::HoeGaarden;
+
+    bb = (BeerBrand)i;  
+    
+    (i < static_cast<int>(BeerBrand::Max) - 1) ? i++ : i = 1;
```

Comment:
Please don't write two expressions in the same line
Also avoid ternary operators (they are hard to read) and also assignment in it (you are using the wrong tool - ternary operator is useful for initialization!).
```suggestion
    if (currentBeerIndex < static_cast<size_t>(BeerBrand::Max) - 1)
    {
        i++;
    }
    else
    {
        currentBeerIndex = 1;
    }
```